SW.Serverless.Samples.CarrierContract/Protos/carrier.proto

```
@@ -0,0 +1,84 @@
+syntax = "proto3";
+
+option csharp_namespace = "SW.Serverless.Samples.CarrierContract";
+package sw.samples.carrier.v1;
```

Buf (1.72.0)

[error] Files with package "sw.samples.carrier.v1" must be within a directory "sw/samples/carrier/v1" relative to root but were in directory "SW.Serverless.Samples.CarrierContract/Protos".

(PACKAGE_DIRECTORY_MATCH)